internal/buf/bufimage/bufimagebuild/bufimagebuildtesting/crashers/115721e379c54d9f8c9055ba554c3c35afce890f

```diff
@@ -0,0 +1,2 @@
+-- .proto --
```

Comment:
Do we need all these files too?

Reply:
I don't think we do. I would rather remove the crashers before merging. I only included them as a way to show the issues the fuzzer found.

Reply:
I've removed them